pom.xml

```diff
@@ -440,6 +440,8 @@
               <excludes>
                 <exclude>@org.apache.hadoop.classification.InterfaceStability$Evolving</exclude>
                 <exclude>@org.apache.hadoop.classification.InterfaceStability$Unstable</exclude>
+                <exclude>com.datatorrent.api.DefaultInputPort</exclude>
```

Comment:
This would open up the ports to any changes. Could we add a comment that after the next release (3.7.0) and setting the compare base version to it, these exclusions need to be removed. It will also be good to have the text in release notes of next release state that the ports have a backward incompatibility change.

Reply:
makes sense. will add a comment in the code as well as on JIRA.